### PR TITLE
cmd/dbcmd: add inspect trie tool

### PR DESCRIPTION
**Pull request by sysvm**
### Description

Add inspect trie tool to analysis the trie tree more convenient.

- https://github.com/bnb-chain/bsc/pull/2082
- https://github.com/bnb-chain/bsc/pull/2142

### Rationale

N/A

### Example

```
 ./geth db inspect-trie --datadir server/data-seed/ latest 4000
```

### Changes

Notable changes:
* N/A

### CODE REVIEW

File: trie/inspect_trie.go [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+		totalNodeCnt := stat.totalNodeStat.ShortNodeCnt + stat.totalNodeStat.ValueNodeCnt + stat.totalNodeStat.FullNodeCnt
+		sortedTriesByNums = append(sortedTriesByNums, SortedTrie{totalNum: totalNodeCnt, ownerAddress: ownerAddress})
+	}
+	sort.Slice(sortedTriesByNums, func(i, j int) bool {
+		return sortedTriesByNums[i].totalNum > sortedTriesByNums[j].totalNum
+	})
+	fmt.Println("EOA accounts num: ", inspect.eoaAccountNums)
+	// only display top 5
+	for i, t := range sortedTriesByNums {
+		if i > 5 {
```
Comment (will-2012) [inline trie/inspect_trie.go:282]: The condition will display the top 6.

**Review by will-2012 — DISMISSED** [review]

**Review by joeylichang — APPROVED** [review]

File: trie/inspect_trie.go (hunk 1 above)
Comment (sysvm) [reply trie/inspect_trie.go:282]: fixed

**Review by krish-nr — APPROVED** [review]
Confirmed with the author that this feature was picked from a previous implementation but was removed in the recent version. For performance monitoring purposes, we are adding it back in this time.

**Review by welkin22 — APPROVED** [review]
LGTM

**Pull request merged**